v0/pipeline/steps/common/provenance-gcr-source-spec.yaml

```diff
@@ -0,0 +1,25 @@
+title: GcrSourceSpec
+allOf:
+- $ref: provenance-source-spec.yaml
+- type: object
+  required:
+  - connector
+  - host
+  - projectID
+  - imageName
+  - digest
+  properties:
+    connector:
+      type: string
+    host:
+      type: string
+    projectID:
```

Comment:
Why aren't we using snakecase?

Reply:
to keep it backaward compatible with are older step which was synonymous with the build and push step